service-front/docker/web/etc/confd/templates/default.conf

```diff
@@ -126,24 +122,7 @@ server {
     #    internal;
     }
 
-    location = /favicon.ico {
```

Comment:
Do we need to keep this line for the favicon?

Reply:
Good catch yes I think we still need it so added back